Packs/WootCloud/Integrations/WootCloud/WootCloud.yml

```diff
@@ -0,0 +1,991 @@
+category: Data Enrichment & Threat Intelligence
+description: Append HyperContext™ insights to your SIEM data and feed them into your
+  orchestration workflows.
+display: WootCloud
+name: WootCloud
+commonfields:
+  id: WootCloud
+  version: -1
+configuration:
+  - display: Client ID
+    name: client_id
+    defaultvalue: ""
+    type: 0
+    required: true
+  - display: API Key
+    name: secret_key
+    defaultvalue: ""
+    type: 4
+    required: true
+  - display: Time to retrieve the first fetch (<number> <time unit>, e.g., 12 hours, 7 days)
+    name: fetch_time
+    required: false
+    type: 0
+    defaultvalue: 3 days
+  - display: Alert Type
+    name: alert_type
+    required: True
+    type: 15
+    options:
+      - packet
+      - bluetooth
+      - anomaly
+  - display: Severity Type
+    name: severity_type
+    required: True
```

Comment:
Should this param be mandatory?

Reply:
Yes, it determines what Severity level to fetch. @mayalaso could go more into the API specifics.

Reply:
Yes this is mandatory field.

Reply:
So Please Change that also

Reply:
Please note my comment here, need to change required to false

Reply:
Change it to not mandatory